RadarSDK/RadarTripOptions.m

```diff
@@ -55,6 +55,8 @@ + (RadarTripOptions *)tripOptionsFromDictionary:(NSDictionary *)dict {
         scheduledArrivalAt = [RadarUtils.isoDateFormatter dateFromString:(NSString *)scheduledArrivalAtObj];
```

Comment:
nit: should these 3 conditions be nested?

might be easier to read / reason about if you're checking the presence of the object first, then checking the type within

it'd also short-circuit slightly quicker when `scheduledArrivalAt` isn't present

Reply:
https://github.com/radarlabs/radar-sdk-ios/pull/264/commits/1a5ae8a297bf0ab0c9c11dc5ec1fca6b770536ed